release notes/v0.44.0.md

```diff
@@ -0,0 +1,224 @@
+- [#2953](https://github.com/grafana/k6/pull/2953). Fixed active VU reporting by arrival-rate executors.
+- [#2833](https://github.com/grafana/k6/pull/2833). Fixed detected memory leaks.
+- [#3006](https://github.com/grafana/k6/pull/3006). `xk6-websockets` updated to v0.2.0 which fixes a memory leak.
+- [xk6-browser#781](https://github.com/grafana/xk6-browser/pull/781) Fix mapping of `response` object's function from `jSON` to `json`.
+- [xk6-browser#779](https://github.com/grafana/xk6-browser/pull/779) Clear Zombie processes on panic.
+- [xk6-browser#834](https://github.com/grafana/xk6-browser/pull/834) Fix `page.close` so that it closes the current page and not the whole browser context.
+
+## Maintenance and internal improvements
+
+- [#2879](https://github.com/grafana/k6/pull/2879). Improved the per-VU buffer pool which should reduce memory usage and improve performance. Thanks, @davidpst!
+- [#3004](https://github.com/grafana/k6/pull/3004). Changed `eventloop.WaitOnRegistered` to execute all scheduled callbacks.
+- [#2881](https://github.com/grafana/k6/pull/2881). Refactored how modules are loaded. This is a preparation for the upcoming ESM support.
+- [#2920](https://github.com/grafana/k6/pull/2920). Updated Go version that we use for k6 compilation to 1.20.x. The docker image is also updated to use alpine:3.17.
+- [#2986](https://github.com/grafana/k6/pull/2986). Refactored goja's `isNullish` to be a part of `js/common`.
+- [#2960](https://github.com/grafana/k6/pull/2960). Refactored `sirupsen/logrus` usage.
+- [#2999](https://github.com/grafana/k6/pull/2999). Directly embed lib.TestPreInitState in js/common.InitEnvironment.
+- [#2931](https://github.com/grafana/k6/pull/2931), [#2940](https://github.com/grafana/k6/pull/2940), [#2895](https://github.com/grafana/k6/pull/2895), [#3002](https://github.com/grafana/k6/pull/3002). Updated k6's dependencies. Added `goja`'s generator support.
+- [#2947](https://github.com/grafana/k6/pull/2947), [#2943](https://github.com/grafana/k6/pull/2943), [#2946](https://github.com/grafana/k6/pull/2946), [#3009](https://github.com/grafana/k6/pull/3009), [#3012](https://github.com/grafana/k6/pull/3012), [#2894](https://github.com/grafana/k6/pull/2894). Tests' fixes, refactoring, and improvements.
+- [#2891](https://github.com/grafana/k6/pull/2891), [#2921](https://github.com/grafana/k6/pull/2921), [#2923](https://github.com/grafana/k6/pull/2923), [#2990](https://github.com/grafana/k6/pull/2990), [#2995](https://github.com/grafana/k6/pull/2995), [#3016](https://github.com/grafana/k6/pull/3016), [#2989](https://github.com/grafana/k6/pull/2989). Linters and formatting fixes.
+- [#3005](https://github.com/grafana/k6/pull/3005). A samples catalog was renamed to examples.
+- [xk6-browser#776](https://github.com/grafana/xk6-browser/pull/776) Fix test for preset flags for chrome on darwin based systems.
```

Comment:
@ankur22 let's try to keep the list of changes shorter to help our users detect what is actually important in the release. For example, if the PR is purely about the testing, we could combine under the `Tests' fixes, refactoring, and improvements.`. What do you think?

Reply:
The xk6-browser changes that are placed under "Other minor changes in this release" were meant to go in "Maintenance and internal improvements", if i moved there, would that be ok?

Reply:
They are still under the headline `## Maintenance and internal improvements`.

The line `Other minor changes in this release` appears since we highlighted `### Improved the per-VU buffer pool`, which has a bigger impact than the other points from the list. 

If it's not clear maybe we could change the sentence to `Other minor maintenance and internal improvements in the release`. WDYT?

Reply:
You could open the preview of the file with the link https://github.com/grafana/k6/blob/0c34d12ad0d9f99ef6df3d3456faf7b9edc7352c/release%20notes/v0.44.0.md

Reply:
I've tried to consolidate some of these in this commit: 458dfb713fe0361eb275846c4feebc3e3b24b9eb